internal/jennies/java/templates/runtime/variants.tmpl

```diff
@@ -0,0 +1,5 @@
+package cog.variants;
+
+public interface {{ .Variant }} {
+    public void Implements{{ .Variant }}Variant();
```

Comment:
Do we need this empty method in Java? Isn't the fact that a class explicitly has to *implement* the interface enough?

Reply:
That's true. I didn't recall it.